tutorials/hess.ipynb

```diff
@@ -86,19 +135,45 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# pos_crab = SkyCoord.from_name('Crab')\n",
-    "pos_crab = SkyCoord(83.633, 22.014, unit=\"deg\")"
+    "model_config = \"\"\"\n",
+    "components:\n",
+    "- name: crab\n",
+    "  type: SkyModel\n",
+    "  spatial:\n",
+    "    type: PointSpatialModel\n",
+    "    frame: icrs\n",
+    "    parameters:\n",
+    "    - name: lon_0\n",
```

Comment:
Can you make an alternative parameter parser work?
```
- lon_0: 83.63 deg
- lat_0: ...
```
This would be shorter and more pleasant to write?
Should be possible to put this in a single place in a generic way for the code that creates `Parameters` from config?